Source/Schema.NET/ValuesJsonConverter.cs

```diff
@@ -309,11 +309,20 @@ private static bool TryProcessTokenAsType(JsonReader reader, Type targetType, ou
                         enumString = valueString;
                     }
 
+#if NETSTANDARD2_0 || NET472 || NET461
+                    try
+                    {
+                        result = Enum.Parse(targetType, enumString);
+                        success = true;
+                    }
+                    catch (Exception)
+#else
                     if (Enum.TryParse(targetType, enumString, out result))

```

Comment:
This seemed like an OK compromise. Not a huge fan of using 3 targets like I am here though.

It did make me wonder, why do we target .NET 4.7.2 and .NET 4.6.1 directly when we also target .NET Standard 2.0?

Reply:
I repeat this type of check here as well with my switched up way to generate a `ToString` for `PropertyValueSpecification`: https://github.com/RehanSaeed/Schema.NET/pull/259/commits/85dde5a5605bd2449358949a6306aaf537cf2d0b#diff-4170f976c7ab6112252917fe8fa2c6a2ce7bed515547e0e42e11ea4d67402f9dR47-R51